NuKeeper.Inspection/Logging/IConfigureLogger.cs

```diff
@@ -0,0 +1,13 @@
+namespace NuKeeper.Inspection.Logging
+{
+    public enum LogDestination
+    {
+        Console,
+        File
+    }
+
+    public interface IConfigureLogger
+    {
+        void Initialise(LogLevel logLevel, string filePath);
```

Comment:
Uhm, that file is a bit odd. Enum isn't related to the interface, and the interface is mixing log levels and file logging.